features/environment.py

```diff
@@ -33,7 +40,18 @@ def before_all(context):
 
     context.login_page = LoginPage(context)
     context.authorized_page = AuthorizedPage(context)
+    context.wait = ui.WebDriverWait(context.driver, 10)
 
 
-def after_all(context):
+def after_scenario(context, scenario):
+    print("scenario status" + scenario.status)
+    if scenario.status == "failed":
+        if not os.path.exists("failed_scenarios_screenshots"):
+            os.makedirs("failed_scenarios_screenshots")
+        os.chdir("failed_scenarios_screenshots")
```

Comment:
Instead of this i can use
 `("failed_scenarios_screenshots/"+scenario.name + get_date_time() + "_failed.png")`
without changing dir. Its will be better?